tests/test_cnes_sonarqube.py

```diff
@@ -0,0 +1,238 @@
+"""Automated integration test of CNES SonarQube
```

Comment:
Codacy found an issue: [Multi-line docstring summary should start at the second line (D213)](https://app.codacy.com/gh/cnescatlab/sonarqube/pullRequest?prid=6252252)